bugsnag-android-core/src/main/java/com/bugsnag/android/ErrorType.kt

```diff
@@ -5,6 +5,11 @@ package com.bugsnag.android
  */
 enum class ErrorType(internal val desc: String) {
 
+    /**
+     * A error captured from Android's JVM layer
+     */
+    EMPTY(""),
```

Comment:
I think we can rather make this a more useful / error related idea, even if the content and use remains the same.

```suggestion
     * An error with an unknown type or source
     */
    UNKNOWN(""),
```